src/dali_model.h

```diff
@@ -91,7 +103,7 @@ class DaliModel : public ::triton::backend::BackendModel {
 
     const char* model_repo_path;
     TRITONBACKEND_ArtifactType artifact_type;
-    TRITON_CALL_GUARD(
+    TRITON_CALL(
```

Comment:
Could you explain why the TRITON_CALL_GUARD -> TRITON_CALL change?

Reply:
TRITON_CALL_GUARD is an old API that should be removed all together. The new TRITON_CALL uses a dedicated exception type and I needed it here.